BUMO/debug.log

```diff
@@ -0,0 +1,2 @@
+[0310/134049:ERROR:tcp_listen_socket.cc(76)] Could not bind socket to 127.0.0.1:6004
```

Comment:
you should add `debug.log` to your `.gitignore` file, this shouldn't be committed.